### PR TITLE
Explorepage

### PR DESCRIPTION
**Pull request by prankurpandeyy**
Things to watch : 

- Sidebar 
- Homepage Hero section
- Videocard with data 
- code quality and UI
- Chips for filtering data

### CODE REVIEW

File: slate-video-library/src/Components/Hero/Hero.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  width: 10rem;
+  background-color: rgb(122, 199, 6);
+  transform: scale(1.2);
+  border: none;
+  margin-left: 3rem;
+}
+
+button.btn.btn-success:hover {
+  background-color: rgb(67, 67, 67);
+  color: white;
```
Comment (JuvitaSaini09) [inline slate-video-library/src/Components/Hero/Hero.css:56]: suggestion:use either rgb or color value for consistency

File: slate-video-library/src/Components/Video-Card/Videocard.js [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
           className="video__cards-iframe-videos"
           title=" videos"
           alt="videodata"
-          src="https://www.youtube.com/embed/tgbNymZ7vqY"
+          src={videoUrl}
         ></iframe>
         <div className="video__cards-details">
           <div className="video__cards-image">
-            <img src="/" alt="images" className="" /> channel title
+            <img src={creator_pic} alt="images" className="" />
```
Comment (JuvitaSaini09) [inline slate-video-library/src/Components/Video-Card/Videocard.js:18]: suggestion:alt should not contain text like picture,image,img etc ,instead you can write "creator"

**Review by JuvitaSaini09 — APPROVED** [review]
It looks good to merge

File: slate-video-library/src/Components/Hero/Hero.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-start;
+  flex-direction: column;
+  margin-top: 10rem;
+}
+button.btn.btn-success {
+  height: 3.5rem;
+  width: 10rem;
+  background-color: rgb(122, 199, 6);
```
Comment (kgajriaAgarwal) [inline slate-video-library/src/Components/Hero/Hero.css:48]: take variables for the colours

**Review by kgajriaAgarwal — COMMENTED** [review]
changes : take variable in root for color.rest everything looks fine , can be merged

**Pull request merged**